test/test_SM_observable_problems.cpp

```diff
@@ -21,12 +22,17 @@ BOOST_AUTO_TEST_CASE( test_non_perturbative_running )
    flexiblesusy::SM_input_parameters input;
    flexiblesusy::SM_mass_eigenstates sm;
    flexiblesusy::Spectrum_generator_settings settings;
-   flexiblesusy::LToLConversion_settings ltolconversion_settings;
    const double scale = 0.0;
 
    setup_SM_const(sm, input);
 
+#if(defined ENABLE_FEYNARTS && defined ENABLE_FORMCALC)
```

Comment:
I'd prefer the syntax
~~~.cpp
#if defined(ENABLE_FEYNARTS) && defined(ENABLE_FORMCALC)
~~~
because it would be consistent across FS (see for example `src/problems.cpp`).